.github/workflows/make_release.yaml

```diff
@@ -40,6 +40,8 @@ jobs:
     name: Get changelog and build the distribution package
     runs-on: ubuntu-latest
     needs: increment-version
+    outputs:
+      changelog: ${{ steps.generate_changelog.outputs.changelog }}
```

Comment:
this is the fix/hack from the issue you linked, so LGTM

Reply:
well, let's not call it a hack! seems to be the right thing to do: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#using-job-outputs-in-a-matrix-job

Reply:
Indeed - you can find I provided the same URL in the commit message 😅

Reply:
that'ts where I got it from 😂 just wanted to clarify